### PR TITLE
Add share to yammer functionality

### PR DESCRIPTION
**Pull request by swordfischer**
Yammer is an enterprise social networking service, and this PR allows to share pages from dokuwiki to yammer.
I use "fa-yammer" even though the icon does not exist - but there are several requests for it (https://github.com/FortAwesome/Font-Awesome/issues/2707) . Thought that would be a better option than not using anything OR using the wrong icon.

### CODE REVIEW

File: js/template.js [hunk 1]
```diff
@@ -695,6 +695,7 @@ jQuery(document).on('bootstrap3:page-icons', function(event) {
     'pinterest'   : (function(){ return [ 'https://pinterest.com/pin/create/button/?url=', url, '&description=', title ].join(''); })(),
     'telegram'    : (function(){ return [ 'https://telegram.me/share/url?url=', url ].join(''); })(),
     'whatsapp'    : (function(){ return [ 'whatsapp://send?text=', title, ': ', url ].join(''); })(),
+    'yammer'      : (function(){ return [ 'https://www.yammer.com/ramboll.com/messages/new?login=true&trk_event=yammer_share&status=', url, '#/Messages/bookmarklet'].join(''); })(),
```
Comment (LotarProject) [inline js/template.js]: The corrected url is `https://www.yammer.com/messages/new?login=true&trk_event=yammer_share&status=`. After the login you will be redirected into your company profile.

File: tpl_page_icons.php [hunk 2]
```diff
@@ -106,6 +106,11 @@
       <li>
         <a href="whatsapp://send?text=" class="share-whatsapp" title="<?php echo tpl_getLang('share_on') ?> Whatsapp" data-action="share/whatsapp/share"><i class="fa fa-fw fa-lg fa-whatsapp"></i> Whatsapp</a>
       </li>
+      <?php endif;
+            if (in_array('yammer', $social_share_providers)): ?>
+      <li>
+        <a href="#" class="share-yammer" title="<?php echo tpl_getLang('share_on') ?> Yammer"><i class="fa fa-fw fa-lg fa-yammer"></i> Yammer</a>
```
Comment (LotarProject) [inline tpl_page_icons.php]: I think the similar and alternative icon is `fa-yahoo`

:P

**Comment by LotarProject** [comment]
Thank you for this PR. I have added a small review to your code!

At your disposal,
Joseph

File: js/template.js (hunk 1 above)
Comment (swordfischer) [reply js/template.js]: Slight oversight on my part, thanks!

**Comment by swordfischer** [comment]
The PR has been updated with requested changes :)

**Comment by LotarProject** [comment]
Thanks !!! :+1:

**Pull request merged**